packages/led_emitter/config/led_emitter_node/LED_protocol.yaml

```diff
@@ -1,14 +1,17 @@
 LED_protocol:
 
   colors:
+    switchedoff: [0,0,0]
+    white: [1,1,1]
     green: [0,1,0]
     red: [1,0,0]
     blue: [0,0,1]
-    white: [1,1,1]
     yellow: [1,1,0]
     purple: [1,0,1]
     cyan: [0,1,1]
-    black: [0,0,0]
+    pink: [1,0,0.5]
+    tl_red: [0,1,0]   # used for the traffic light LEDs as they are differently ordered
+    tl_green: [1,0,0] # used for the traffic light LEDs as they are differently ordered
```

Comment:
Isn´t it confusing to have multiple times the same color?
Also if we use more colors on TL we would have double the entries

Reply:
The problem comes from the different ordering of the RGB channels on a bot and a tower. Before, the traffic light node loaded this LED_protocol (which is not part of its configuration) and modified the colors in it. In my opinion it was dirty and non-modular solution. This is also not ideal, but I do not foresee additional colors on a traffic light (maybe yellow but nothing more).
Alternatively, we can have an additional parameter that controls the channel order, and on launch time, depending to whether we launch it on a bot or a tower, we use a different value for it. What do you think?